community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WhereTest.scala

```diff
@@ -32,8 +32,8 @@ class WhereTest extends DocumentingTestBase {
 
   override val properties = Map(
     "Andres" -> Map[String, Any]("age" -> 36l, "belt" -> "white"),
-    "Tobias" -> Map[String, Any]("age" -> 25l),
-    "Peter"  -> Map[String, Any]("age" -> 34l)
+    "Tobias" -> Map[String, Any]("age" -> 25l, "address" -> "Sweden/Malmö"),
```

Comment:
I would argue that there is a character set bug here that should be fixed. Either we saved the Malmö in the wrote character set, or the graph renderer is not using UTF8.